recipes/pyright/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "pyright" %}
+{% set version = "0.0.10" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pyright-{{ version }}.tar.gz
+  sha256: c3c8303442ddd39003100dcc892542b1a7f38fcbb131360784a9b1ea6f8ddccf
+
+build:
+  number: 0
+  entry_points:
```

Comment:
`setup.py` lists [two more entry points](https://github.com/RobertCraigie/pyright-python/blob/20ba327b53f82bcab283060896fbca3d93d8a936/setup.py#L50-L53). Is that alright? Just making sure you didn't miss them on accident!

Reply:
Not on this version; while this PR is waiting 2 new releases were made upstream; those entrypoints were added after